packages/@tinacms/graphql/src/primitives/resolver/index.ts

```diff
@@ -530,8 +529,9 @@ export class Resolver {
       case 'string':
         if (field.options) {
           if (field.list) {
+            // FIXME: this is awaiting checkbox suppport
             return {
-              component: 'checkbox-group',
+              component: 'checkbox',
```

Comment:
I'm not quite sure this PR is up-to-date with the latest changes to `tinacms`?  I believe my PR to convert this to `checkbox-group` was merged in last week?

Reply:
Good eye @Enigmatical.

There must have been some oddities locally that made their way into that file, because I do not recall touching those parts of the file. 

Either way, those changes have been reverted.